### ISSUE TITLE
Tailwind component is broken on codesandbox.io

### ISSUE THREAD

**Issue by soultoru** [resendlabs/react-email#940]
### Describe the Bug

I tried demo scripts on codesandbox. but, I can't get rendered html with Tailwind component.

### Which package is affected (leave empty if unsure)

@react-email/tailwind

### Link to the code that reproduces this issue

https://codesandbox.io/s/react-email-forked-z5shzl

### To Reproduce

copy code from [demo](https://demo.react.email/preview/vercel-invite-user), and paste codesandbox

![Screenshot_20230927_143110](https://github.com/resendlabs/react-email/assets/1527724/e3685106-5536-4358-8e5d-bc90af32ca3a)

### Expected Behavior

module is not being read correctly

**Comment by bjcarlson42** [comment]
I am experiencing this too. [Downgrading](https://socket.dev/npm/package/@react-email/tailwind/overview/0.0.8) to version `0.0.8` is a temporary fix.

**Comment by shellscape** [comment]
I [commented yesterday](https://github.com/resendlabs/react-email/pull/923#issuecomment-1735564669) on the PR that @bukinoshita merged, that the change had broken tailwind rendering on my fork. (which if you're interested in using, ping me). They need to do a revert and republish.

**Comment by gabrielmfern** [comment]
Testing the same code locally doesn't reproduce the error,. Seems more like a problem with CodePen or something about the environment rather than a problem with react-email. Also the changes in #923 don't really seem like they would cause a problem with importing of a module.

**Comment by AlecRust** [comment]
Related, I'm also having a similar issue in my Next.js app when upgrading `@react-email/tailwind` 0.0.8 to 0.0.9:

From the server logs:

```
 ⨯ file:///Users/alec/projects/personal/my_project/node_modules/@react-email/tailwind/dist/index.mjs:29
import { tailwindToCSS } from "tw-to-css";
         ^^^^^^^^^^^^^
SyntaxError: Named export 'tailwindToCSS' not found. The requested module 'tw-to-css' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'tw-to-css';
const { tailwindToCSS } = pkg;
```

**Comment by soultoru** [comment]
I tried from v0.0.1 to v0.0.9 on codesandbox.
- success : 0.0.2, 0.0.3
- failure: others

**Comment by gabrielmfern** [comment]
> I tried from v0.0.1 to v0.0.9 on codesandbox.
> 
>     * success : 0.0.2, 0.0.3
> 
>     * failure: others

I am wondering if this is more a problem with `tw-to-css`'s cjs/esm interoperability and support, I saw a few issues open about it.

**Comment by gabrielmfern** [comment]
I've made a [PR](https://github.com/vinicoder/tw-to-css/pull/11) on `tw-to-css` that should've fixed this, now we just need to wait to be merged.

**Comment by gabrielmfern** [comment]
Seems like this is still a problem after the PR was merged into `tw-to-css`, probabbly due to a small mistake when building and then publishing it, but this should be fixed after #1024 since we won't rely on `tw-to-css` anymore due to perf reasons.
